wadas/domain/custom_classification_mode.py

```diff
@@ -0,0 +1,71 @@
+"""Custom Classification module."""
+
+import logging
+from queue import Empty
+
+from wadas.domain.animal_detection_mode import AnimalDetectionAndClassificationMode
+from wadas.domain.camera import img_queue
+
+logger = logging.getLogger(__name__)
+
+
+class CustomClassificationMode(AnimalDetectionAndClassificationMode):
+    """Custom Classification Mode class."""
+
+    def __init__(self, target_animal_label="bear"):
```

Comment:
We have to check for target animal passed as parameter. it shall match with the list available in the model. (See wadas/ai/models.py list)